api-catalog-ui/frontend/src/actions/wizard-actions.jsx

```diff
@@ -18,6 +18,7 @@ import {
     REMOVE_INDEX,
     VALIDATE_INPUT,
     UPDATE_SERVICE_ID,
+    UPDATE_UPLOADED_YAML_TITLE,
```

Comment:
this new action needs to be tested too, in the wizard-actions.test.jsx, as reported by the coverage analysis